packages/shared-ux/button_toolbar/README.mdx

```diff
@@ -1,9 +1,11 @@
 ---
-id: sharedUX/ButtonToolbar
-slug: /shared-ux/button-toolbar
-title: Button Toolbar
-description: 
-tags: ['shared-ux', 'component']
-date: 2022-06-14
+id: sharedUX/Components/Toolbar
+slug: /shared-ux/components/button_toolbar
+title: Toolbar
+summary: An implementation of the popover, primary button, icon button group and add from library button
+tags: ['shared-ux', 'component', 'toolbar']
+date: 2022-07-28
 ---
 
+This `toolbar` component is a component that requires the prop `children`. The `children` prop can include the `popover`, previously primary now generic `button`, `icon button group` and `add from library button` for solutions. 
```

Comment:
I find this sentence a bit confusing. I think it's meant to say children can include a popover and a generic button to trigger that button. It can also include icon button group and "add from library" button for solutions.

Reply:
Good call - I think I should also include an example for clarity thanks! [3247796](https://github.com/elastic/kibana/pull/137224/commits/324779666ce1e58346ae84b3ef6ac4d08d7ecdbc)

Reply:
Much better, thanx!